tools/runner

```diff
@@ -96,16 +96,17 @@ try:
             param_name = param.group(1).lower()
             param_value = param.group(2)
 
-            if param_name not in req_test_params:
+            if (param_name not in req_test_params):
                 logger.warning(
                     "Unsupported test param found: {} - ignoring".format(
                         param_name))
                 continue
 
             test_params[param_name] = param_value
 
+            # check all items in the req_test_params exists in the test_params.
             if len(set(req_test_params) - set(test_params.keys())) == 0:
-                # all parameters found
+                # all required parameters found
                 break
```

Comment:
Probably we should rename the `req_test_params` to something like `supported_test_params` and modify the remaining code/comments accrodingly. It's name/comment suggests that those are all required but in fact this is just a list of all parameters we look for/support which can get confusing.

Reply:
yes that's what confused me at the beginning, it said required, so when i needed an optional tag i created one for it